src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -655,17 +655,25 @@ public String[] getApplicationProtocols() {
      * and stored as a byte-oriented {@code String} before calling this method.
      *
      * <blockquote><pre>
-     *     // MEETEI MAYEK LETTERS HUK UN I (Unicode 0xabcd->0xabcf): 2 bytes
-     *     byte[] bytes = "\u005cuabcd\u005cuabce\u005cuabcf"
-     *             .getBytes(StandardCharsets.UTF_8);
-     *     String HUK_UN_I = new String(bytes, StandardCharsets.ISO_8859_1);
+     *     // Encode 3 Meetei Mayek letters (HUK, UN, I) using Unicode Escapes
+     *     //     0xabcd->0xabcf, 2 Unicode bytes/letter.
+     *     String HUK_UN_I =  "\u005cuabcd\u005cuabce\u005cuabcf";
      *
-     *     // 0x00-0xFF:  1 byte
-     *     String rfc7301Grease8A = "\u005cu008A\u005cu008A";
+     *     // Convert into UTF-8 encoded bytes (3 bytes/letter)
+     *     byte[] bytes = HUK_UN_I.getBytes(StandardCharsets.UTF_8);
      *
+     *     // Convert bytes into a ISO_8859_1 encoded string
+     *     String encodedHukUnI =
+     *         new String(bytes, StandardCharsets.ISO_8859_1);
+     *
```

Comment:
Maybe a better comment here would be:
```
    *      // Preserve bytes by using ISO_8859_1 encoding
```
because we're not converting anything into anything. 

As far as I understand what you are trying to do is to shoe-horn a byte array into a string, so that you can retrieve the original array using `String.getBytes(StandardCharsets.ISO_8859_1)`?